p256/point.ml

```diff
@@ -9,26 +9,103 @@ let at_infinity () =
   let f_z = Fe.create () in
   {f_x; f_y; f_z}
 
+let fe_from_be_cstruct cs =
+  let cs_rev = Cstruct.rev cs in
+  let fe = Fe.create () in
+  Fe.from_bytes fe cs_rev; Fe.to_montgomery fe; fe
+
+let is_on_curve ~x ~y =
+  let a =
```

Comment:
should we extract a `Parameters` module?

Reply:
That might be a good idea since we'll probably need them elsewhere too (eg the curve order for #2)

Reply:
done (I put the generator there as well)